src/components/Dashboard.vue

```diff
@@ -568,7 +606,7 @@ export default {
 }
 
 
-@media only screen and (max-width: 1347px) {
+@media only screen and (max-width: 878px) {
```

Comment:
Same comment as above

Reply:
Similar reasoning as to why this.isTablet = window.innerwidth <= 878 and not 976 from up above